src/jit/codegenarmarch.cpp

```diff
@@ -730,20 +741,52 @@ void CodeGen::genPutArgStk(GenTreePutArgStk* treeNode)
                 structOffset += (2 * TARGET_POINTER_SIZE);
                 nextIndex += 2;
             }
+#else  // _TARGET_ARM_
+            // For a >= 4 byte structSize we will generate a ldr and str instruction each loop
+            //             ldr     r2, [r0]
+            //             str     r2, [sp, #16]
```

Comment:
I think it's not important what the offset value is. It is just example. The offset value isn't offset of this struct argument. It means stack offset in all of arguments for method.